Palace/Network/Reachability.swift

```diff
@@ -0,0 +1,73 @@
+//
+//  Reachability.swift
+//  Palace
+//
+//  Created by Vladimir Fedorov on 02/02/2023.
+//  Copyright © 2023 The Palace Project. All rights reserved.
+//
+
+import Network
+import SystemConfiguration
+
+// TODO: Replace TPPReachability and TPPReachabilityManager with this class
+@objcMembers
+class Reachability: NSObject {
```

Comment:
We have `TPPReachability` and `ReachabilityManager` classes, but they work by checking the availability of `www.apple.com`. This class uses modern methods of the `Network` framework.